File: 1816-truncate-sentence/1816-truncate-sentence.js
/**
 * @param {string} s
 * @param {number} k
 * @return {string}
 */
var truncateSentence = function(s, k) {
    
    let result = "";
    const dividedString = s.split(" ");

    for(i = 0; i < k; i++){
        result +=  " " + dividedString[i];
    }

    return result.trim();
};

File: 1816-truncate-sentence/1816-truncate-sentence.java
class Solution {
    public String truncateSentence(String s, int k) {
        
        String result = "";
        String[] divideString = s.split(" ");

        for(int i = 0; i < k; i++){
            result = result + " " + divideString[i];
        }

        
        return result.trim();
    }
}